### ISSUE TITLE
Peer dependencies

### ISSUE THREAD

**Issue by thenickcox** [ntdb/quickstart#4]
On running `npm install`.

```
npm WARN eslint-config-airbnb@7.0.0 requires a peer of eslint-plugin-jsx-a11y@^0.6.2 but none was installed.
npm WARN babel-eslint@5.0.4 requires a peer of eslint@<2.3.0 but none was installed.
```

It just looks like a warning, but running `npm start` fails with this:

```
Module build failed: Error: Failed to load plugin jsx-a11y: Cannot find module 'eslint-plugin-jsx-a11y'
Referenced from: airbnb
Referenced from: /Users/user/git/elephant/.eslintrc
...
```

**Comment by thenickcox** [comment]
This may actually be the result of the failed install of `eslint-config-airbnb`.

**Comment by thenickcox** [comment]
```
Listening at localhost:3000:
webpack: wait until bundle finished: /assets/bundle.js
webpack: wait until bundle finished: /assets/bundle.js

ERROR in ./source/index.js

/Users/user/git/elephant/source/index.js
  1:1  error  Definition for rule 'jsx-a11y/img-uses-alt' was not found     jsx-a11y/img-uses-alt
  1:1  error  Definition for rule 'jsx-a11y/redundant-alt' was not found    jsx-a11y/redundant-alt
  1:1  error  Definition for rule 'jsx-a11y/valid-aria-role' was not found  jsx-a11y/valid-aria-role
```
